jecs.d.ts

```diff
@@ -289,6 +289,9 @@ export function pair_first<P, O>(world: World, p: Pair<P, O>): Entity<P>;
  */
 export function pair_second<P, O>(world: World, p: Pair<P, O>): Entity<O>;
 
+export function ECS_PAIR_FIRST<P, O>(pair: Pair<P, O>): Entity<P>;
+export function ECS_PAIR_SECOND<P, O>(pair: Pair<P, O>): Entity<O>;
```

Comment:
This is a bit of a slippery slope in my eyes. `pair_first` and `pair_second` returns valid entities but these functions do not. They basically just give you the lower/higher bits respectively which could return a valid entity id or not! E.g. `pair(ComponentA, ComponentB)` where A and B are low numbers like `5` and `11` respectively, this would be fine. But if you put larger entities with generations then they would not be able to retrieve the fully entity. My use cases basically just need them as an index rather than as a valid entity so these should probably return numbers.